workloads/whereabouts/run.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash -e
+
+set -e
+
+ES_SERVER=${ES_SERVER=https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com}
+LOG_LEVEL=${LOG_LEVEL:-info}
+if [ "$KUBE_BURNER_VERSION" = "default" ]; then
+    unset KUBE_BURNER_VERSION
+fi
+KUBE_BURNER_VERSION=${KUBE_BURNER_VERSION:-1.1.0}
+CHURN=${CHURN:-false}
+WORKLOAD='node-density'
+QPS=${QPS:-20}
+BURST=${BURST:-20}
+GC=${GC:-true}
+EXTRA_FLAGS=${EXTRA_FLAGS:-}
+UUID=${UUID:-$(uuidgen)}
+KUBE_DIR=${KUBE_DIR:-/tmp}
+export ITERATIONS=${ITERATIONS:-341}
+
+download_binary(){
+  KUBE_BURNER_URL="https://github.com/kube-burner/kube-burner-ocp/releases/download/v${KUBE_BURNER_VERSION}/kube-burner-ocp-V${KUBE_BURNER_VERSION}-linux-x86_64.tar.gz"
+  curl --fail --retry 8 --retry-all-errors -sS -L "${KUBE_BURNER_URL}" | tar -xzC "${KUBE_DIR}/" kube-burner-ocp
+}
+function cleanup_whereabouts(){
+    
+    # remove IP pool
+    oc delete ippools.whereabouts.cni.cncf.io '10.1.0.0-21' -n openshift-multus
+
+    overlaps=$(oc get overlappingrangeipreservations.whereabouts.cni.cncf.io -n openshift-multus --no-headers=true | awk '{print $1}');
+
+    # also need to remove the overlapping reservations
+    for i in $overlaps; do
+        oc delete overlappingrangeipreservations.whereabouts.cni.cncf.io $i -n openshift-multus;
+    done
+
+    # This will remove the whereabouts plugin from the network operator. The operator will stop the whereabouts pods on its own
+    oc patch network.operator.openshift.io cluster --type=json -p='[{"op":"remove","path":"/spec/additionalNetworks"}]'
+
+
+}
+function install_whereabouts(){
+
+    # Install whereabouts reconciler daemon
+    oc patch network.operator.openshift.io cluster --patch-file=reconciler.yml --type=merge
+    exit_code=$?
+    if [ $exit_code -eq 0 ]; then
+        echo "whereabouts reconciler daemon installed via oc patch"
+    else
+        echo "failure - reconciler daemon not installed"
+        exit $exit_code
+    fi
+    # give the pods time to come up.
+    sleep_time=10
+    echo "sleeping for $sleep_time seconds to allow reconciler pods to start"
+    sleep $sleep_time
+}
+
+
+hypershift(){
```

Comment:
Are we doing anything with hypershift -- i realize you might have just taken the existing `run.sh` but lets clean it up to be more specific for this work.